### ISSUE TITLE
[FE] Update for address validation where there is only one option

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#4632]
## User Story:

As a person using a screenreader, I need the CTAs to be clearer when I am trying to update my address, address validation is triggered, and there is only one address to save. 

## Background

Copy ticket #4529

### Override allowed; no suggestions

When override is allowed and there are no suggestions — including for the use cases where someone needs to add a unit number or edit a bad unit number, please change these screens as follows:

- Remove the radio button next to the entered address
- Update the main (blue) CTA button copy to "Use this address"

![13  Scenario 2_ Override; No suggestions](https://user-images.githubusercontent.com/34068740/71928135-ef546e80-3164-11ea-8cb1-be98ea2284a1.png)

![Add missing unit number with override](https://user-images.githubusercontent.com/34068740/71928140-f24f5f00-3164-11ea-85d6-202928d74704.png)

![Edit bad unit number with override](https://user-images.githubusercontent.com/34068740/71928146-f4192280-3164-11ea-8332-92764876213a.png)

### No override, one suggestion

When override is not allowed and there is only **one** suggestion, please change this screen as follows:

- Remove the radio button next to the suggested address
- Update the main (blue) CTA button copy to "Use suggested address"

![18  Scenario 3_ No override; Only one suggestion](https://user-images.githubusercontent.com/34068740/71928172-fe3b2100-3164-11ea-8dd2-2a1057155aef.png)

**Note**: We will still use the existing pattern if there are at least two or more suggestions:

![18  Scenario 3_ No override; Suggestions](https://user-images.githubusercontent.com/34068740/71928212-1874ff00-3165-11ea-97cc-ff950c03b3cc.png)

## Tasks
- [ ] Remove radio buttons as indicated
- [ ] Update button copy

## Acceptance Criteria:
- Designs are updated and validated (if possible) on staging

**Comment by Samara-Strauss** [comment]
@bradl3yC I went to check on this work because it was in the validate column and saw this. The "Edit address" button should be "Use suggested address" and should save the suggested address. Did I test this too early, or is something else wrong?

<img width="443" alt="Screen Shot 2020-01-13 at 3 46 14 PM" src="https://user-images.githubusercontent.com/34068740/72292037-a007b580-361e-11ea-8e67-1bd167a7d126.png">

**Comment by Samara-Strauss** [comment]
Documenting discussion from Slack — we figured out what was going on. Going from no override + suggestion --> no override, no suggestion --> no override + suggestion was causing the bug above. @bradl3yC knows what's going on and is opening a PR to fix this. Will validate once that bug fix is deployed.

**Comment by Samara-Strauss** [comment]
Tracking fix in #4837

**Comment by Samara-Strauss** [comment]
Closing. This work is done and the saving issue was fixed in #4837.
